pages/landing/enterprise-eks/index.html

```diff
@@ -0,0 +1,277 @@
+    body: |
+      Gruntwork takes your requested configuration and generates Terraform code for deploying your EKS cluster in a
+      repository of your choice. We support integrating with GitHub, GitLab, and BitBucket. Rely on battle-tested
+      infrastructure code that has been proven in production at hundreds of companies and is maintained and supported by
+      DevOps experts. The code we deliver is 100% owned by you - use it as is, or customize to your needs.
+    logos:
+      - title: GitHub
+        src: /assets/img/landing-heroku-page/github-icon.png
+      - title: GitLab
+        src: /assets/img/landing-heroku-page/gitlab-icon.png
+      - title: BitBucket
+        src: /assets/img/landing-heroku-page/bitbucket-icon.png
+
+  - heading: "Step 3: Deploy your EKS cluster"
+    image: /assets/img/landing-page-enterprise/tfc-screenshot.png
+    image_side: right
+    body: |
+      Use the provided Gruntwork Pipelines configuration embedded in the generated code to automatically deploy without
+      installing anything on your machine. Or deploy the code yourself using any of our supported tools:
+      <ul>
+          <li>Terraform</li>
+          <li>Terraform Cloud/Enterprise</li>
+          <li>Terragrunt</li>
+      </ul>
+    logos:
+      - title: Terraform
+        src: /assets/img/landing-heroku-page/terraform-icon.png
+      - title: TFC/TFE
+        src: /assets/img/logos-technologies/icon-hashicorp.png
+      - title: Terragrunt
+        src: /assets/img/logos-technologies/terragrunt.png
+
+  - heading: "Step 4: Configure your apps"
+    image: /assets/img/landing-page-enterprise/sample-app-clip.gif
+    image_side: left
+    body: |
+      Use Gruntwork tools to optimize your enterprise apps for EKS. Our tool will automatically your apps with
+      cloud-native best practices for packaging, monitoring, logging, service discovery, secrets management, schema
+      migrations, and more. We support common enterprise languages and frameworks:
+      <ul>
+          <li>Java: Spring Boot, Quarkus</li>
+          <li>Ruby: Ruby on Rails, Sinatra</li>
+          <li>Python: Django, Flask</li>
+          <li>Node.js: Express.js, Next.js</li>
+      </ul>
+      Or use our enterprise-grade reference applications as a template to implement the optimizations for any language
+      and framework we don't support.
+    logos:
+      - title: Spring Boot
+        src: /assets/img/logos-technologies/logo-spring-boot.png
+      - title: Rails
+        src: /assets/img/logos-technologies/logo-rails.png
+      - title: Django
+        src: /assets/img/logos-technologies/logo-django.png
+      - title: Express
+        src: /assets/img/logos-technologies/logo-express.png
+
+  - heading: "Step 5: Deploy your apps"
+    image: /assets/img/landing-heroku-page/github-step-1.png
+    image_side: right
+    body: |
+      Connect your Git repo to automatically register, migrate, and deploy your apps using a friendly UI. Gruntwork
```

Comment:
Good point. Originally I was thinking it would link to our apps, but after thinking it through, probably makes more sense that it links directly to ArgoCD. Replaced text to clarify that it links to Argo, and updated screenshot to be from Argo.